docs/deploy.md

```diff
@@ -48,12 +48,12 @@ OpenWhisk used to support both shared state and a sharding model. The former has
 
 The sharding loadbalancer has the caveat of being limited in its scalability in its current implementation. It uses "horizontal" sharding, which means that the slots on each invoker are evenly divided to the loadbalancers. For example: In a system with 2 loadbalancers and invokers which have 16 slots each, each loadbalancer would get 8 slots on each invoker. In this specific case, a cluster of loadbalancers > 16 instances does not make sense, since each loadbalancer would only have a fraction of a slot above that. The code guards against that but it is strongly recommended not to deploy more sharding loadbalancers than there are slots on each invoker.
 
-# Invoker use of docker-runc
+# Invoker use of runc
 
-To improve performance, Invokers attempt to maintain warm containers for frequently executed actions. To optimize resource usage, the action containers are paused/unpaused between invocations.  The system can be configured to use either docker-runc or docker to perform the pause/unpause operations by setting the value of the environment variable INVOKER_USE_RUNC to true or false respectively. If not set, it will default to true (use docker-runc).
+To improve performance, Invokers attempt to maintain warm containers for frequently executed actions. To optimize resource usage, the action containers are paused/unpaused between invocations.  The system can be configured to use either runc or docker to perform the pause/unpause operations by setting the value of the environment variable INVOKER_USE_RUNC to true or false respectively. If not set, it will default to true (use runc).
 
-Using docker-runc obtains significantly better performance, but requires that the version of docker-runc within the invoker container is an exact version match to the docker-runc of the host environment.  Failure to get an exact version match will results in error messages like:
+Using runc obtains significantly better performance, but requires that the version of runc within the invoker container is an exact version match to the runc of the host environment.  Failure to get an exact version match will results in error messages like:
```

Comment:
Is this still a requirement with runc that versions must be an exact match? Makes it very hard to use runc in serverless environment where you may control the invoker container but not the underlying server.

Reply:
I think it should not be an exact match, but some versions have breaking changes.
For example, the command's name is changed from `docker-runc` to `runc`.
The root volume directory was also different in the previous version.
https://github.com/apache/openwhisk/pull/4430/files#diff-072209721097df0ae37ac99015b28844fec1f9b390314782671445b4f80af622R185

I assume if there is no such breaking change, runc would be compatible, but anyway, it would be better to use the same version if possible.